code/modules/shuttle/emergency.dm

```diff
@@ -181,6 +181,8 @@
 
 /obj/machinery/computer/emergency_shuttle/proc/increase_hijack_stage()
 	var/obj/docking_port/mobile/emergency/shuttle = SSshuttle.emergency
+	if(!shuttle.hijack_status) // Begin loading this early, otherwise we get an awkward 30 second delay on the 
+		SSmapping.lazy_load_template(LAZY_TEMPLATE_KEY_NUKIEBASE)
```

Comment:
```suggestion
	// Begin loading this early, otherwise we get an awkward 30 second delay on the 
	SSmapping.lazy_load_template(LAZY_TEMPLATE_KEY_NUKIEBASE)
```

lazy_load_template only loads once on its own